cloudformation/cfncluster.cfn.json

```diff
@@ -2943,9 +2864,9 @@
             "AssociatePublicIpAddress" : {
               "Fn::If": [
                 "ComputePublicIps",
-                true,
-                false
-              ],
+                "true",
```

Comment:
I think this was included from a previous commit. Looks like launch template commit.

Reply:
The launch template commit fixed a similar issue under "EbsOptimized", but I think this is a separate issue